src/components/MoneyRequestConfirmationList.js

```diff
@@ -544,15 +556,16 @@ function MoneyRequestConfirmationList(props) {
                             onPress={() => Navigation.navigate(ROUTES.MONEY_REQUEST_DISTANCE.getRoute(props.iouType, props.reportID))}
                             disabled={didConfirm || props.isReadOnly || !isTypeRequest}
                         />
-                    ) : (
+                    )}
+                    {!props.isDistanceRequest && !props.receiptPath && (
```

Comment:
Same here, why do we only want to show the merchant when there's no receipt? I think we still display merchant for regular scans

Reply:
Hmm check this conversation https://expensify.slack.com/archives/C02NZ2HGJAZ/p1696448674463469?thread_ts=1696442783.829229&cid=C02NZ2HGJAZ, it seems like we only want to display the 'Date' and 'Merchant' fields in manual requests

Reply:
@luacmartins we shouldn't show the `date` and `merchant` fields in the `Scan` flow. That's what SmartScan is going to populate, so it would be redundant to let those fields be set AND then initiate SmartScan..

Reply:
Sounds good. Thanks for clarifying!